src/components/Form.js

```diff
@@ -35,34 +35,44 @@ const StyledHeader = styled.h2`
   font-weight: 700;
   color: ${brandColours.senary};
   line-height: 55px;
-  padding: 20px;
+  margin: 20px;
```

Comment:
Remove all the margins and paddings from the header and the form, you just need padding on the StyledInner, where the bg of green is, to create that space, then it's managed once instead of twice in this case.